.gitlab-ci.yml

```diff
@@ -198,6 +198,8 @@ docs:
         -p ink_primitives -p ink_prelude
         -p ink_lang -p ink_lang_macro -p ink_lang_ir -p ink_lang_codegen
     - mv ${CARGO_TARGET_DIR}/doc ./crate-docs
+    # FIXME: remove me after CI image gets nonroot
```

Comment:
I guess this is a comment for the CI/CD team? Should we create a follow-up?

Reply:
it's mentioned both in the issue and description